packages/perseus-core/src/widgets/table/table-util.ts

```diff
@@ -0,0 +1,13 @@
+import type {PerseusTableWidgetOptions} from "@khanacademy/perseus-core";
+
+type TablePublicWidgetOptions = Pick<
+    PerseusTableWidgetOptions,
+    "headers" | "rows" | "columns"
+>;
+
+export default function getTablePublicWidgetOptions(
+    options: PerseusTableWidgetOptions,
+): TablePublicWidgetOptions {
+    const {answers: _, ...publicOptions} = options;
```

Comment:
Random question: if a new key is added to `PerseusTableWidgetOptions` that is answer data, will type checking still allow this spread to pass (incorrectly)?

It appears so: https://www.typescriptlang.org/play/?#code/C4TwDgpgBA8mwEsD2A7AzlAvFA3gKCkKkWABsIAuKNYAJwRQHMAaKAogQ3QHcJaqa9Js3aFRUCAA86HAII8+AugxZQA9GqgA5AKIB1NgF88eUJCgAFAK4AjUggDGcRKgzYLjgNYAeZ8nSsAEQk5IEAfCYAZlYoDi4oUIwQwNZ2jn6uABRI8P5oVBnoAJRUqfZOua644g6uwLhcaLz8UAD6rAB0XWC25YVohlhQOfFoANxs4rTJVrQJPWkVo3iGQA

Reply:
Type checking will pass, but the test will fail.